docs/reference/technology/connectivity/cellular.md

```diff
@@ -48,33 +48,51 @@ If you use an Mbed OS target and a separate cellular hosted module via a serial
         [
             "CELLULAR_DEVICE=<manufacturer-module>",
             "MDMRXD=<rx-pin-name>",
-            "MDMTXD=<tx-pin-name>"
+            "MDMTXD=<tx-pin-name>",
+            "MDMRTS=<rts-pin-name>",
+            "MDMCTS=<cts-pin-name>"
         ]
-    }    
+    }
 
-You need to change the pin-names above to actual pins, such as D0 and D1, according to your Mbed target. You may also need to define MDMRTS and MDMCTS pins if you have RTS/CTS connected on UART.
+You need to change the pin names above to actual pins, such as D0 and D1, according to your Mbed target. You may also need to define MDMRTS and MDMCTS pins if you have RTS and CTS connected on UART. If RTC and CTS are not connected on UART, then define MDMRTS and MDMCTS as `NC`.
 
 ### Cellular APIs
 
-As an application developer, you should use and refer only to classes located under API folder. All the other classes have implementation details which are expected to change frequently.
+As an application developer, you should use and refer only to classes located under API folder. All the other classes have implementation details that are expected to change frequently.
 
 Cellular APIs are structured based on main functionalities:
 
-- `CellularNetwork` for cellular network features, such as preferred operator and APN.
+- `CellularContext` is the main interface for the application. You can use it to connect to the operator's APN.
```

Comment:
Query: What does APN stand for?

Reply:
Access Point Name: https://en.wikipedia.org/wiki/Access_Point_Name